_data-prepper/log-analytics.md

```diff
@@ -0,0 +1,193 @@
+
+The main changes you need to make are:
+
+* `hosts` - Set to your hosts
+* `index` - Change this to the OpenSearch index you want to send logs to
+* `username`- Provide the OpenSearch username
+* `password` - Provide your OpenSearch password
+* `aws_sigv4` - If you use Amazon OpenSearch Service with AWS signing, set this to true. It will sign requests with the default AWS credentials provider.
+* `aws_region` - If you use Amazon OpenSearch Service with AWS signing, set this value to your region.
+
+## FluentBit
+
+You will have to run FluentBit in your service environment. You can find the installation guide of FluentBit [here](https://docs.fluentbit.io/manual/installation/getting-started-with-fluent-bit). Ensure that you can configure the [FluentBit http output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/http) to your Data Prepper Http Source. Below is an example `fluent-bit.conf` that tails a log file named `test.log` and forwards it to a locally running Data Prepper's http source, which runs by default on port 2021. 
+
+Note that you should adjust the file `path`, output `Host` and `Port` according to how and where you have FluentBit and Data Prepper running.
+
+### Example fluent-bit.conf without SSL and basic authentication enabled
+
+Example `fluent-bit.conf` without SSL and Basic Authentication enabled on the http source:
+
+```
+[INPUT]
+  name                  tail
+  refresh_interval      5
+  path                  test.log
+  read_from_head        true
+
+[OUTPUT]
+  Name http
+  Match *
+  Host localhost
+  Port 2021
+  URI /log/ingest
+  Format json
+```
+
+If your http source has SSL and Basic Authentication enabled, you will need to add the details 
+of `http_User`, `http_Passwd`, `tls.crt_file`, and `tls.key_file` to the `fluent-bit.conf` as shown below.
+
+### Example fluent-bit.conf with SSL and basic authentication enabled
+
+Example `fluent-bit.conf` with SSL and Basic Authentication enabled on the http source:
+
+```
+[INPUT]
+  name                  tail
+  refresh_interval      5
+  path                  test.log
+  read_from_head        true
+
+[OUTPUT]
+  Name http
+  Match *
+  Host localhost
+  http_User myuser
+  http_Passwd mys3cret
+  tls On
+  tls.crt_file /full/path/to/certfile.crt
+  tls.key_file /full/path/to/keyfile.key
+  Port 2021
+  URI /log/ingest
+  Format json
+```
+
+# Next Steps
+
+Follow the [Log ingestion demo guide](../examples/log-ingestion/README.md) to go through a specific example of Apache log ingestion from `FluentBit -> Data Prepper -> OpenSearch` running through Docker.
+
+In the future, Data Prepper will contain additional sources and processors which will make more complex log analytic pipelines available. Check out our [Roadmap](https://github.com/opensearch-project/data-prepper/projects/1) to see what is coming.  
```

Comment:
I think we don't need these last two paragraphs.

Reply:
@dlvenable I can remove these last two paragraphs, but that removes the link to the Log ingestion demo guide in the previous comment above. Do you still want to delete both paragraphs here?

Reply:
No, just remove starting at line 190: "In the future, Data Prepper will..."